Correct Reolink battery camera wake frequency (#42634)
home-assistant/home-assistant.github.io: source/_integrations/reolink.markdown

### Battery drains fast

> The Reolink Home Assistant integration is supposed to only wake battery cameras once per 6 hours for a few seconds or perform a data update when the battery camera wakes by itself (at most once per hour). This should not have a big impact on battery life. You can check correct operation using the **Sleep status** entity. However, there are several factors that can have significant impact on battery life:

Reviewer: ```suggestion
The Reolink Home Assistant integration is supposed to wake battery cameras only once every 6 hours for a few seconds, or to perform a data update when the battery camera wakes up on its own (at most once per hour). This should not have a significant impact on battery life. You can check the correct operation using the **Sleep status** entity. However, several factors can have a significant impact on battery life:
```
